tensorboard/data/experimental/experiment_from_dev.py

```diff
@@ -22,6 +22,7 @@
 import numpy as np
```

Comment:
This import is unused now; mind removing it and the placeholder `expect_numpy_installed` dep from the BUILD target?  That should fix the presubmit check.  If you re-request my review after that's updated, I can re-trigger the presubmits.

Reply:
Done!